frontend/src/views/Game.tsx

```diff
@@ -1,31 +1,68 @@
 import Editor from '@monaco-editor/react';
 import React, { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { Problem, getProblems } from '../api/Problem';
 import { Room } from '../api/Room';
+import { Text, ProblemHeaderText } from '../components/core/Text';
+import Header from '../components/navigation/Header';
 
 type LocationState = {
   room: Room,
 }
 
-function GamePage() {
+const FlexContainer = styled.div`
+  display: flex;
+  top: 50px;
+`;
+
+const FlexPanel = styled.div`
+  flex: 1;
+  padding: 1rem;
+  background-color: #e3e3e3;
+  overflow: none;
+  height: 100vh;
+  width: 50vw;
+`;
+
+export default function GamePage() {
   const location = useLocation<LocationState>();
   const [room, setRoom] = useState<Room | null>(null);
 
+  const [problems, setProblems] = useState<Problem[] | null>(null)
+
   // Called every time location changes
   useEffect(() => {
     if (location && location.state && location.state.room) {
       setRoom(location.state.room);
     }
+    getProblems().then((response) => {
+      setProblems(response);
+    }).catch((error) => {
+      console.log(error);
+    });
   }, [location]);
 
+  const firstProblem = problems?.[0];
+
   return (
     <div>
       Room:
       {' '}
       {room ? room.roomId : 'No room joined'}
       <Editor height="90vh" language="javascript" />
+      <div>
+        <Header />
+        <FlexContainer>
+          <FlexPanel>
+            <ProblemHeaderText>{ firstProblem?.name }</ProblemHeaderText>
+            <Text>{ firstProblem?.description }</Text>
+          </FlexPanel>
+          <FlexPanel>
+            <Editor height="100vh" language="javascript" />
```

Comment:
The Editor only fills the space within its `FlexPanel` component; this works even with other FlexBox components occupying the page as shown by the new `RoomInfoBar` component.

Reply:
Sorry, I think I didn't clarify -- IMO the page design should match [LeetCode](https://leetcode.com/problems/single-number-ii/), where you cannot scroll any further on the page (just within the divs, but the bottom of the page remains the footer with things like "Console" and "Submit"). I was asking if you could construct this so it would match that layout; essentially, so you cannot scroll on the whole page.

Reply:
Oh I see what you mean. The page should fit the browser window now.